_ingest-pipelines/processors/remove.md

```diff
@@ -30,7 +30,8 @@ The following table lists the required and optional parameters for the `remove`
 
 | Parameter  | Required/Optional  | Description  |
 |---|---|---|
-`field`  | Required  | The name of the field containing the data to be removed. Supports [template snippets]({{site.url}}{{site.baseurl}}/ingest-pipelines/create-ingest/#template-snippets). The following metadata fields are not allowed to be removed: `_index`, `_version`, `_version_type`, and `_id`. Note that `_id` is not allowed to be removed when there's a specified external version for the ingesting document. |
+`field`  | Optional  | The field name containing the data to be removed. Supports [template snippets]({{site.url}}{{site.baseurl}}/ingest-pipelines/create-ingest/#template-snippets). The following metadata fields cannot be removed: `_index`, `_version`, `_version_type`, and `_id`. If a version is specified for the ingesting document, the `_id` cannot be removed. |
```

Comment:
As written, "ingesting document" implies that the document is ingesting something. Is this accurate, or should we reword?

Reply:
Revised: `The metadata fields `_index`, `_version`, `_version_type`, and `_id` cannot be removed. If `version` is specified, `_id` cannot be removed from the ingested document.`